Cargo.toml

```
@@ -28,6 +28,7 @@ subprocess = "0.2"
 indexmap = "1"
 format_serde_error = "0.3"
 regex = "1"
+cmd_lib = "1.1.0"
```

```suggestion
cmd_lib = "1"
```

When a crate is already available as 1.x.x, you can safely limit the version constraint to the major version, because any changes to the minor or patch versions are compatible.